src/brainglobe_napari/atlas_viewer_utils.py

```diff
@@ -0,0 +1,15 @@
+import json
+from pathlib import Path
+
+from bg_atlasapi.list_atlases import get_local_atlas_version
+
+
+def read_atlas_metadata_from_file(atlas_name: str):
+    """Reads atlas metadata stored in a `.json` in the BrainGlobe directory."""
+    brainglobe_dir = Path.home() / ".brainglobe"
```

Comment:
Ideally this wouldn't be hardcoded, as it can be set by the user. See e.g. https://github.com/brainglobe/bg-atlasapi/blob/b6e955df1b88b6a998cb4653cc2d76ab57911126/bg_atlasapi/bg_atlas.py#L67

Reply:
@adamltyson just to double-check: are you suggesting we let users call this function with additional optional arguments like so
```python
def read_atlas_metadata_from_file(atlas_name: str, brainglobe_dir=None, interim_download_dir=None, config_dir=None):
```

Reply:
Or only some of them?

Reply:
Moved this discussion to #28 - will merge for now.

Reply:
Sounds good. I only meant that the atlases may not be stored in that directory, if the user has changed it previously.